CIFAR10/train/ryan_resnet164_nb_1p100-dawn_tensorflow.tsv

```diff
@@ -0,0 +1,182 @@
+23.02208	0.4861799916293886	81.62
+24.02304	0.5072788535224066	82.91
+25.024	0.5283645629220539	83.73
+26.02496	0.5494618541002274	81.92
+27.02592	0.5705832342969047	81.28999999999999
+28.02688	0.5916673221853045	81.17
+29.02784	0.6127595403459337	84.91
+30.0288	0.6338588282134798	83.57
+31.02976	0.6549512921439277	80.32000000000001
+32.03072	0.6760326563649707	82.78999999999999
+33.03168	0.6971264672941632	77.62
+34.03264	0.718198007411427	81.2
+35.0336	0.7393069404363631	82.47
+36.03456	0.7603817120525572	81.2
+37.03552	0.7815092153019375	83.74000000000001
+38.03648	0.8026095634036594	80.65
+39.03744	0.8237145894765854	84.5
+40.0384	0.8448203788863288	81.67
+41.03936	0.8659089588456684	82.91
+42.04032	0.8870152540339364	81.92
+43.04128	0.9080939974387487	83.24000000000001
+44.04224	0.929165239599016	83.25
+45.0432	0.9502241923411687	80.72
+46.04416	0.9713082849317126	86.15
+47.04512	0.9923841693666247	80.72
+48.04608	1.0134796379672157	85.24000000000001
+49.04704	1.0345600505007637	85.45
+50.048	1.0556601793898477	85.48
+51.04896	1.0767608091566296	83.82
+52.04992	1.0978533584541743	84.55
+53.05088	1.1189415286646949	85.57000000000001
+54.05184	1.1400370658768548	82.78999999999999
+55.0528	1.1611190219720204	84.61999999999999
+56.05376	1.1821986247433558	86.36
+57.05472	1.2033038526773452	85.6
+58.05568	1.2243887103266187	86.56
+59.05664	1.2454683112435871	82.13000000000001
+60.0576	1.2665350025229984	82.32000000000001
+61.05856	1.2875984148846733	86.03
+62.05952	1.30868614534537	84.14
+63.06048	1.3297359169191785	82.73
+64.06144	1.3508021643426682	81.38
+65.0624	1.3718834465742111	85.61
+66.06336	1.3929506213797465	83.22
+67.06432	1.4140134655104744	86.71
+68.06528	1.4350932970311907	83.89999999999999
+69.06624	1.456149232784907	85.52
+70.0672	1.4772126093175677	86.00999999999999
+71.06816	1.49830287012789	85.63
+72.06912	1.519400901860661	85.28999999999999
+73.07008	1.540471566187011	86.14
+74.07104	1.5615409907367495	85.76
+75.072	1.5826025736331941	86.06
+76.07296	1.603668161564403	83.55
+77.07392	1.6247225759426753	85.14
+78.07488	1.6458013400766585	85.02
+79.07584	1.6668743936883077	83.54
+80.0768	1.6879427301221424	84.98
+81.07776	1.7089948996570377	86.88
+82.07872	1.7300665270619924	84.6
+83.07968	1.751141729619768	87.15
+84.08064	1.7721827832195494	85.61
+85.0816	1.7932581733995014	86.53
+86.08256	1.8143254182073807	84.15
+87.08352	1.8354008897807863	84.96000000000001
+88.08448	1.8564790541595884	85.95
+89.08544	1.8775458999474843	86.22999999999999
+90.0864	1.898606893022855	85.61999999999999
+91.08736	1.9196692901849746	87.01
+92.08832	1.9407238294680913	92.33
+93.08928	1.9617921262979507	92.60000000000001
+94.09024	1.9828753029637867	92.81
+95.0912	2.0039472704463535	92.84
+96.09216	2.0250190403064092	92.99
+97.09312	2.0461121825377147	92.72
+98.09408	2.067193559077051	92.97
+99.09504	2.0882425423463187	92.69
+100.096	2.109299462040265	92.95
+101.09696	2.1303626803557076	92.75
+102.09792	2.1514113063282436	93.05
```

Comment:
@lxylzxc we are only reviewing results that are in the [top-5 results for at least one leaderboard](https://dawn.cs.stanford.edu/2018/07/25/dawnbench_rolling/). It doesn't look like this is fast enough to be considered for training in terms of time or cost. Can you remove the CIFAR10/train files from this pull request?

Reply:
thanks for  your reply, I will remove the CIFAR10/train files.